data/players/.gitignore

```diff
@@ -1,2 +1,3 @@
 /*_friends.json
 /*_loadouts.json
+/backups
```

Comment:
Update for new backup names

Reply:
Backup directories are now ignored in the top level gitignore.